mosviz/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = 'mosviz' %}
+{% set version = '0.0.1' %}
+{% set tag = 'v0.0.1' %}
```

Comment:
You don't need to change this, but here's a useful trick to avoid typing the version twice:
```cfg
{% set tag = 'v' + version %}
```